src/utiltime.cpp

```diff
@@ -25,7 +28,7 @@ int64_t GetTime()
 
 void SetMockTime(int64_t nMockTimeIn)
 {
-    nMockTime = nMockTimeIn;
+    nMockTime.store(nMockTimeIn, std::memory_order_relaxed);
```

Comment:
GetTime is called all over the place, no? I don't want to introduce extra synchronization there.